android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/UriBeaconDiscoveryService.java

```diff
@@ -477,12 +477,13 @@ private RemoteViews updateSummaryNotificationRemoteViews() {
   private void updateSummaryNotificationRemoteViewsFirstBeacon(String url, RemoteViews remoteViews) {
     MetadataResolver.UrlMetadata urlMetadata_firstBeacon = mUrlToUrlMetadata.get(url);
```

Comment:
Since you're already touching this part of the code. Would you mind changing the name of the variable.

https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s5.2.7-local-variable-names